README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
 
-[![Build Status](https://travis-ci.org/shopware/platform.svg?branch=6.1)](https://travis-ci.org/shopware/platform)
```

Comment:
could you revert this change?

Reply:
I don´t know how to revert just one file, how should i do it?